wiki/Contests/Skinning_Contest/2/en.md

```diff
@@ -45,12 +47,43 @@ The Skinning Contest #2 is run by various community members.
 - [Announcement newspost](https://osu.ppy.sh/home/news/2021-12-02-skinning-contest-tides-of-winter-announcement)
+- [Voting Open newspost](https://osu.ppy.sh/home/news/2022-01-16-skinning-contest-tides-of-winter-voting-open)
+- [Results out newspost](https://osu.ppy.sh/home/news/2022-02-25-skinning-contest-tides-of-winter-results)
 - [Discussion thread](https://osu.ppy.sh/community/forums/topics/1472845)
 - [Submission thread](https://osu.ppy.sh/community/forums/topics/1472846)
 - [Discord server](https://discord.skinship.xyz)
 - [Official website](https://skinship.xyz)
 
+## Participants
+
+| Skinner | Entry |
+| :-- | :-- |
+| ![][flag_PL] [Redo_](https://osu/ppy/sh/users/7122165) | [- 『Wintherest』 -](https://osu.ppy.sh/community/forums/topics/1498493) |
+| ![][flag_VN] [TKieen](https://osu/ppy/sh/users/12561202) | [105°C Chocolate](https://osu.ppy.sh/community/forums/topics/1496067) |
+| ![][flag_BY] [thetasigma](https://osu/ppy/sh/users/6234482) | [Azure Snowfall](https://osu.ppy.sh/community/forums/topics/1498492) |
+| ![][flag_VN] [koomo](https://osu/ppy/sh/users/2168518) | [CACTUS'](https://osu.ppy.sh/community/forums/topics/1481190) |
+| ![][flag_PH] [BPChingu](https://osu/ppy/sh/users/4236855) | [GanyuTeks:Futur Céleste](https://osu.ppy.sh/community/forums/topics/1489926) |
+| ![][flag_FR] [Ootsu](https://osu/ppy/sh/users/9634575) | [HotChocolate](https://osu.ppy.sh/community/forums/topics/1484895) |
+| ![][flag_PH] [Inflique](https://osu/ppy/sh/users/13865585) | [Inflique's Snow Machine](https://osu.ppy.sh/community/forums/topics/1498697) |
+| ![][flag_ID] [ArchieA7](https://osu/ppy/sh/users/7087699) | [Musim Dingin](https://osu.ppy.sh/community/forums/topics/1480386) |
+| ![][flag_BR] [JoSanInk](https://osu/ppy/sh/users/10848682) | [N-ice](https://osu.ppy.sh/community/forums/topics/1497539) |
+| ![][flag_JP] [too](https://osu/ppy/sh/users/12196931) | [Shirokuma シロクマ](https://osu.ppy.sh/community/forums/topics/1476743) |
+| ![][flag_ID] [Akshiro](https://osu/ppy/sh/users/10557490) | [Snow](https://osu.ppy.sh/community/forums/topics/1494794) |
+| ![][flag_FR] [Corne2Plum3](https://osu/ppy/sh/users/15646039) | [Snowy Circles](https://osu.ppy.sh/community/forums/topics/1498442) |
+| ![][flag_ID] [nayuu](https://osu/ppy/sh/users/12561379) | [snowyy](https://osu.ppy.sh/community/forums/topics/1498720) |
+| ![][flag_MX] [ZiskyST](https://osu/ppy/sh/users/11844975) | [The Lost Winter](https://osu.ppy.sh/community/forums/topics/1494537) |
+| ![][flag_US] [Icysandwich](https://osu/ppy/sh/users/10961544) | [Tides of osu!](https://osu.ppy.sh/community/forums/topics/1498620) |
+| ![][flag_CA] [RUDEKA](https://osu/ppy/sh/users/13015586) | [TOWN](https://osu.ppy.sh/community/forums/topics/1484572) |
+| ![][flag_MY] [HIDOKOMORI123](https://osu/ppy/sh/users/14216890) | [WINTERMACHI](https://osu.ppy.sh/community/forums/topics/1491283) |
+
+## Podium
+
+*For full results, see the [contest page](https://osu.ppy.sh/community/contests/137).*
```

Comment:
```suggestion
*For the full results, see the [contest page](https://osu.ppy.sh/community/contests/137).*
```

Reply:
I simply copied this from the first contests page, i guess it should also be changed there?

Reply:
yes